README.md

```diff
@@ -5,8 +5,8 @@
 [wmf5-blog-post]: https://msdn.microsoft.com/en-us/powershell/wmf/releasenotes
 [wmf5-blog-incompatibilites]: https://msdn.microsoft.com/en-us/powershell/wmf/productincompat
 
-
 #### Table of Contents
+
 1. [Description - What is the dsc module and what does it do](#module-description)
 2. [Prerequisites](#windows-system-prerequisites)
 3. [Setup](#setup)
```

Comment:
The bulleted items are subsections of the numbered items. This is standard across our READMEs, so it's fine.